typescript/package.json

```diff
@@ -17,7 +17,7 @@
     "scripts": {
         "start": "webpack serve --mode development --devtool inline-source-map --entry ./src/demo/index.js --open",
         "typecheck": "tsc --noEmit",
-        "lint": "eslint *.js *.json \"packages/*/src/**/*.{js,jsx,ts,tsx,mdx,json}\"",
+        "lint": "eslint *.js *.json \"packages/*/*.{js,jsx,ts,tsx,mdx,json}\" \"packages/*/src/**/*.{js,jsx,ts,tsx,mdx,json}\"",
```

Comment:
Starting to get complex. Why not check all and use .eslintignore for exceptions.

Reply:
Done